src/components/ui/Banner.vue

```diff
@@ -14,30 +14,52 @@
     />
 
     <div class="banner-text">
-      Metamask currently does not support signatures using Ledger/Trezor devices
+      Attention! As per
+      <a
+        href="https://forum.abracadabra.money/t/aip-12-increase-weth-and-wbtc-cauldrons-interest/3992"
+        target="_blank"
+        class="banner-link"
+        >AIP #12</a
+      >, all interest fees for this cauldron will be applied to your collateral
+      rather than your borrowed amount.
     </div>
-
-    <BaseButton :width="'112px'"> Button </BaseButton>
   </div>
 </template>
 
 <script>
-const BaseButton = () => import("@/components/base/BaseButton");
 export default {
   data() {
     return {
-      isBanner: false,
+      routes: ["BorrowId", "RepayId", "LeverageId", "DeleverageId"],
+      ids: ["28", "27"],
+      closeClicked: false,
     };
   },
 
+  computed: {
+    chainId() {
+      return this.$store.getters.getChainId;
+    },
+    showBanner() {
+      if(this.chainId !== 1) return false;
```

Comment:
same thing here, this should a per-chain customization configuration, which is read upon app loading / refreshed when the chain changes